tests/test_ebuild.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from superflore.generators.ebuild.ebuild import Ebuild
+from superflore.generators.ebuild.ebuild import ebuild_keyword
+from superflore.exceptions import UnresolvedDependency
+import unittest
+
+
+class TestEbuildOutput(unittest.TestCase):
+    @classmethod
```

Comment:
You don't want classmethod here. That's for things like additional constructors. https://stackoverflow.com/questions/12179271/meaning-of-classmethod-and-staticmethod-for-beginner not just a regular method inside a class.https://docs.python.org/3/library/functions.html#classmethod

Reply:
That makes sense. I was afraid that was some `unittest`-induced magic. With that definition, the context in which I had previously seen it makes more sense now.